docs_user/modules/proc_migrating-mon-from-controller-nodes.adoc

```diff
@@ -2,394 +2,12 @@
 
 = Migrating Ceph Monitor daemons to {Ceph} nodes
 
-You must move Ceph Monitor daemons from the {rhos_prev_long} ({OpenStackShort}) Controller nodes to a set of target nodes. Target nodes are either existing {Ceph} nodes, or {OpenStackShort} Compute nodes if {Ceph} is
-deployed by {OpenStackPreviousInstaller} with a Hyperconverged Infrastructure (HCI) topology.
+include::./proc_migrating-mon-from-controller-nodes-prerequisites.adoc[leveloffset=+1]
 
-To migrate the Ceph Monitor daemons, the following actions occur:
+include::./proc_migrating-mon-from-controller-nodes-drain-host.adoc[leveloffset=+1]
```

Comment:
@klgill o/ this is my idea and attempt of the `Ceph Mon` procedure split.
An alternative approach can be to move this `proc_migrating-mon-from-controller-nodes.adoc` `module` into an `assembly`, but it might result confusing because the current structure is:

```
ceph-rbd-migration-assembly
|-> ceph-mgr-migration-main-procedure
|-> ceph-mon-migration-main-procedure
     |-> ceph-mon-migration-prerequisite-procedure
     |-> ceph-mon-migration-drain-procedure
     |-> ceph-mon-migration-network-procedure
     |-> ceph-mon-migration-redeploy-procedure
     |-> ceph-mon-migration-verification-procedure
```

which means that we keep _procedures_ into modules (and not mixing things), and we make it modular.
If it helps we can create subdirectories within `modules/`, so you can easily find under `modules/ceph/mon` the right content, but I didn't propose that pattern because I'm not sure how do you feel about that.
Let's do a first pass of this proposal and we'll refine with multiple iterations.

Reply:
@fmount I think we're going to need an assembly to group all of the "ceph mon" procedures together. Can I check out your branch and move things around/make edits?

Reply:
I'm ok to have an assembly for the main `mon` procedure that groups together multiple procedures:

```
|-> ceph-mon-migration-main-procedure   -> Assembly
     |-> ceph-mon-migration-prerequisite-procedure  -> proc
     |-> ceph-mon-migration-drain-procedure  -> proc
     |-> ceph-mon-migration-network-procedure  -> proc
     |-> ceph-mon-migration-redeploy-procedure  -> proc
     |-> ceph-mon-migration-verification-procedure  -> proc
 ```
 
 Being that said feel free to checkout the branch and make the edits you need, and we can review it in the next pass.

Reply:
For some reason I can't check out your branch. I get " error: pathspec 'ceph_doc' did not match any file(s) known to git". I'll try to make the edits in the PR.

Reply:
mm weird, the branch is remote and is here [1], not on this repository (but in my fork).
You can try the following and see if it helps:

```bash
git remote add francesco https://github.com/fmount/data-plane-adoption
git remote -v # it shows the new remote
git fetch francesco
git checkout -b ceph_doc francesco/ceph_doc

# do some edits to the files

git push francesco ceph_doc:ceph_doc
```
The last command should be able to push to my branch and trigger an update to this PR.

[1] https://github.com/fmount/data-plane-adoption/tree/ceph_doc